core/wiki/config/TagsMinLength.tid

```diff
@@ -0,0 +1,3 @@
+title: $:/config/Tags/MinLength
+
+0
```

Comment:
w/o a limit on the results, 0 is not a practical option at all and should at least be 1